mitiq/calibration/tests/test_calibration.py

```diff
@@ -4,11 +4,13 @@
 # LICENSE file in the root directory of this source tree.
 
 """Tests for the Clifford data regression top-level API."""
+import re
 from functools import partial
 
 import cirq
 import numpy as np
 import pytest
+from mock import MagicMock, call
```

Comment:
According to https://pypi.org/project/mock/ we should be able to import these from `unittest` which will allow us to avoid adding a dependency!

Reply:
Updated to use `unittest`